pkg/volume/util/metrics.go

```diff
@@ -50,7 +50,7 @@ var storageOperationMetric = metrics.NewHistogramVec(
 	[]string{"volume_plugin", "operation_name", "status"},
 )
 
-var storageOperationStatusMetric = metrics.NewCounterVec(
+var StorageOperationStatusMetric = metrics.NewCounterVec(
```

Comment:
I have not found any criterion to decide the package visibility of such metric variables. But this should be OK according to similar ones in kubelet and scheduler.

https://github.com/kubernetes/kubernetes/blob/bff6452893e58e269290fe321c21285244ecb95f/pkg/kubelet/metrics/metrics.go#L86-L94

https://github.com/kubernetes/kubernetes/blob/bff6452893e58e269290fe321c21285244ecb95f/pkg/scheduler/metrics/metrics.go#L98-L107